benches/payout-stakers/src/main.rs

```diff
@@ -127,10 +169,25 @@ fn nominate_validator(
         .for_each(|chunk| staking_batch_nominate(connection, chunk));
 }
 
-fn set_validators(address: &str) -> Vec<KeyPair> {
-    let validators = (0..VALIDATOR_COUNT)
-        .map(|validator_number| derive_user_account(validator_number))
-        .collect::<Vec<_>>();
+fn set_validators(
+    address: &str,
+    validators_seed_file: Option<String>,
+    validators_count: u32,
+) -> Vec<KeyPair> {
+    let validators = match validators_seed_file {
+        Some(validators_seed_file) => {
+            let validators_seeds = std::fs::read_to_string(&validators_seed_file)
+                .expect(&format!("Failed to read file {}", validators_seed_file));
+            validators_seeds
+                .split("\n")
+                .filter(|seed| !seed.is_empty())
+                .map(keypair_from_string)
+                .collect()
+        }
+        None => (0..validators_count)
+            .map(|validator_number| derive_user_account_from_numeric_seed(validator_number))
+            .collect::<Vec<_>>(),
+    };
```

Comment:
feels like this should already be handled before calling the set_validators function.

Reply:
right